matchzoo/models/dense_baseline_model.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Why this line here?

Reply:
If we remove this line, it will not pass the checking of flake8. If you run make init & make test, you will find this problem.

Reply:
This file has been here for a long time. If anything weird happens you should first let its developer know instead of directly making a change. The problem could be your flake version, your local flake rules e.t.c. not necessarily the code. Plus, adding a new line is not the best solution.

Reply:
Fixed